src/Symfony/Component/Console/Formatter/OutputFormatter.php

```diff
@@ -157,7 +158,7 @@ public function formatAndWrap(?string $message, int $width)
             $offset = $pos + \strlen($text);
 
             // opening tag?
-            if ($open = '/' != $text[1]) {
+            if ($open = ('/' !== $text[1])) {
```

Comment:
I added it because PHPStorm recommended this change, and I agree, it is more readable what is happening, even if you are totally right, the original version worked. If you want, I can 'undo' this, up to you. But now you know why I did it.

Reply:
Let's revert as it's not needed.